src/common/utils/api/api.ts

```diff
@@ -120,11 +126,24 @@ interface AuthTestResponse {
   username: string;
 }
 
+export interface SuccessResponse {
```

Comment:
The name kind of indicates that the response will be always success? But I guess it can be also false?

Reply:
Yes, maybe PermissionResponse?

Reply:
That sounds better imo

Reply:
Rename'd